autopts/bot/common_features/report.py

```diff
@@ -13,24 +13,25 @@
 # more details.
 #
 
+import datetime
 import logging
 import os
 import re
-import datetime
-import zipfile
 import shutil
+import zipfile
 from pathlib import Path
+from typing import List, Optional
 from xmlrpc.client import ServerProxy
 
 import git
-import yaml
 import xlsxwriter
+import yaml
```

Comment:
e.g. exception for sorted includes is seen here where it goes `import` -> `from` -> `import` -> `from autopts`